src/nnet3bin/nnet3-copy.cc

```diff
@@ -87,11 +87,11 @@ int main(int argc, char *argv[]) {
 
     if (learning_rate >= 0)
       SetLearningRate(learning_rate, &nnet);
-    
+
     if (scale != 1.0)
       ScaleNnet(scale, &nnet);
-    
-    if (dropout > 0)
+
+    if (dropout >= 0)
```

Comment:
I believe we're removing these dropout options from the scripts.
[however, for this code to work as intended, the default value of 'dropout' should be -1, not zero].
Anyway, we should remove all related options and code and rely on the --edits and --edits-config options.